.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1 @@
+For questions, now we can use https://github.com/kubeedge/kubeedge/discussions
```

Comment:
This doesn't block people from creating questions as issues.
You may refer to: https://github.com/kubernetes/kubernetes/blob/master/.github/ISSUE_TEMPLATE/config.yml

Reply:
Good suggestion.

Reply:
fix